metrics/lib/features/dashboard/presentation/widgets/expandable_text.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+/// The widget that resizes [text] to fill all available space if
+/// wrapped by [Flexible] widgets.
+class ExpandableText extends StatelessWidget {
```

Comment:
What do you think if we change the name to "FittedText"?

Reply:
I think 'ExpandableText' name describes the widget behavior. This widget should expand text, but not apply some fit. Therefore, I suggest to leave this name and change the 'resizes' in documentation to 'expands' to make it more clear.